pinot-controller/src/main/java/org/apache/pinot/controller/BaseControllerStarter.java

```diff
@@ -661,6 +665,28 @@ protected TaskManagerStatusCache<TaskGeneratorMostRecentRunInfo> getTaskManagerS
     return new InMemoryTaskManagerStatusCache();
   }
 
+  protected List<PinotSegmentDeletionListener> getSegmentDeletionListeners() {
```

Comment:
Let's move this into the `SegmentDeletionManager`. We don't want to change the signature for `PinotHelixResourceManager.start()`

Reply:
I've moved it to `PinotHelixResourceManager` since I needed to initialize the listeners with HelixManager. LMK if that's ok